dist/k8s/staging/frontend.yml

```diff
@@ -0,0 +1,34 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: frontend
+  labels:
+    app: frontend
+spec:
+  selector:
+    matchLabels:
+      app: frontend
+  template:
+    metadata:
+      labels:
+        app: frontend
+    spec:
+      containers:
+      - name: frontend
+        image: docker.pkg.github.com/wirvsvirus-gemeinsam-sport/gemeinsam-sport-frontend/frontend:0.0.61
```

Comment:
Same Question. stage/staging/dev

Reply:
this one is automatically patched by the github action... but actually I probably broke that using this... the `sed` replaced :latest with this... So will fix it.

Reply:
fixed